react-multiple-pages/my-app/src/pages/NotFound.tsx

```diff
@@ -0,0 +1,12 @@
+export function NotFound() {
+  return (
+    <div className="flex">
+      <div className="flex-1 py-12 text-center">
+        <h3>Uh oh, we could not find the page you were looking for!</h3>
+        <div className="text-gray-700 cursor-pointer">
```

Comment:
This needs to be a Link, also. Otherwise "Return to Dashboard" doesn't actually do anything!